README.md

```diff
@@ -31,6 +31,18 @@ For running some specific tests you can do this as following:
 
     $ python -m unittest tests.test_publisher
 
+
+Installation
+------------
+
+Install `skt` directly from git:
+
+    $ pip install git+https://github.com/RH-FMK/skt
```

Comment:
Could you also add a paragraph about "installation" for development, from already checked-out repo?

Reply:
Can do.